app-admin/rcm/rcm-9999.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit git-r3
+
+DESCRIPTION="rc file (dotfile) management"
+HOMEPAGE="https://github.com/thoughtbot/rcm"
+EGIT_REPO_URI="https://github.com/thoughtbot/rcm.git"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS=""
+
+DEPEND="dev-ruby/mustache"
+RDEPEND="${DEPEND}"
+
+src_prepare () {
+	./autogen.sh || die
```

Comment:
I'd recommend to swap these two lines, just to be nice to any potential user that applies user patches - this will ensure that autogen happens afterwards.